src/Core/Domain/Category/Command/AbstractEditCategoryCommand.php

```diff
@@ -0,0 +1,302 @@
+     *
+     * @return $this
+     */
+    public function setLocalizedDescriptions(array $localizedDescriptions): self
+    {
+        $this->localizedDescriptions = $localizedDescriptions;
+
+        return $this;
+    }
+
+    /**
+     * @return bool|null
+     */
+    public function isActive(): ?bool
+    {
+        return $this->isActive;
+    }
+
+    /**
+     * @param bool $isActive
+     *
+     * @return $this
+     *
+     * @throws CategoryConstraintException
+     */
+    public function setIsActive(bool $isActive): self
+    {
+        if (!is_bool($isActive)) {
+            throw new CategoryConstraintException('Invalid Category status supplied', CategoryConstraintException::INVALID_STATUS);
+        }
+
+        $this->isActive = $isActive;
+
+        return $this;
+    }
+
+    /**
+     * @return string[]|null
+     */
+    public function getLocalizedMetaTitles(): ?array
+    {
+        return $this->localizedMetaTitles;
+    }
+
+    /**
+     * @param string[] $localizedMetaTitles
+     *
+     * @return $this
+     */
+    public function setLocalizedMetaTitles(array $localizedMetaTitles): self
+    {
+        $this->localizedMetaTitles = $localizedMetaTitles;
+
+        return $this;
+    }
+
+    /**
+     * @return string[]|null
+     */
+    public function getLocalizedMetaDescriptions(): ?array
+    {
+        return $this->localizedMetaDescriptions;
+    }
+
+    /**
+     * @param string[] $localizedMetaDescriptions
+     *
+     * @return $this
+     */
+    public function setLocalizedMetaDescriptions(array $localizedMetaDescriptions): self
+    {
+        $this->localizedMetaDescriptions = $localizedMetaDescriptions;
+
+        return $this;
+    }
+
+    /**
+     * @return string[]|null
+     */
+    public function getLocalizedMetaKeywords(): ?array
+    {
+        return $this->localizedMetaKeywords;
+    }
+
+    /**
+     * @param string[] $localizedMetaKeywords
+     *
+     * @return $this
+     */
+    public function setLocalizedMetaKeywords(array $localizedMetaKeywords): self
+    {
+        $this->localizedMetaKeywords = $localizedMetaKeywords;
+
+        return $this;
+    }
+
+    /**
+     * @return int[]|null
+     */
+    public function getAssociatedGroupIds(): ?array
+    {
+        return $this->associatedGroupIds;
+    }
+
+    /**
+     * @param int[] $associatedGroupIds
+     *
+     * @return $this
+     */
+    public function setAssociatedGroupIds(array $associatedGroupIds): self
+    {
+        $this->associatedGroupIds = $associatedGroupIds;
+
+        return $this;
+    }
+
+    /**
+     * @return int[]|null
+     */
+    public function getAssociatedShopIds(): ?array
+    {
+        return $this->associatedShopIds;
+    }
+
+    /**
+     * @param int[] $associatedShopIds
+     *
+     * @return $this
+     */
+    public function setAssociatedShopIds(array $associatedShopIds): self
+    {
+        $this->associatedShopIds = $associatedShopIds;
```

Comment:
```suggestion
        $this->associatedShopIds = array_map('intval', $associatedShopIds);
```

Or something similar because nothing say it will contaéins only ids?

Reply:
Yeah array map makes sense thanks!